### ISSUE TITLE
Loss keeps oscillating during training

### ISSUE THREAD

**Issue by haikunzhang95** [clovaai/cutblur#26]
Hi！

I use the network I designed for training, but the loss keeps oscillating during the training process. I tried different learning rates, but it didn't work. Do you have any suggestions?

Thank you！
